spec/services/discovery_report_spec.rb

```diff
@@ -117,12 +116,12 @@
       )
     end
 
-    it 'produces the json, indicates if errors occurs and records the error message' do
+    it 'produces the json, indicates no errors occurred' do
       json = JSON.parse(report.to_builder.target!)
       expect(json['rows'].size).to eq 3
-      expect(report.objects_had_errors).to be true
-      expect(report.error_message).to eq '2 objects had errors in the discovery report'
-      expect(report.summary).to include(objects_with_error: 2, total_size: 636_563)
+      expect(report.objects_had_errors).to be false
+      expect(report.error_message).to be_nil
+      expect(report.summary).to include(objects_with_error: 0, total_size: 382_224)
     end
```

Comment:
there are no errors in the discovery report for these fixture objects -- so the expectations can be adjusted